src/lib/preview-data.ts

```
@@ -0,0 +1,946 @@
+            id: Date.now() + i,
+            type,
+            user_id: user.id,
+            user_name: `${user.primer_nombre} ${user.primer_apellido}`,
+            timestamp
+        };
+        
+        switch (type) {
+            case 'post_new':
+                const post = posts[Math.floor(Math.random() * posts.length)];
+                activity.post_id = post.id;
+                activity.post_title = post.titulo;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)